### PR TITLE
feat: add links related to open food facts in settings route.

### PR DESCRIPTION
**Pull request by Sudhanva-Nadiger**
### What
- Add download links for different platforms
- Add donate link
- Add join community links and discover projects links
- Add a footer with all social-media links

### Screenshot
image
image

### Fixes bug(s)
- #101 

### Part of 
- #37

### CODE REVIEW

**Comment by Sudhanva-Nadiger** [comment]
@alexfauquette I got confused about where to add the share link of hunger games! if u explain to me that I will ad that too! Thanks.

File: src/pages/settings/index.jsx [hunk 1]
```diff
@@ -50,95 +52,103 @@ export default function Settings() {
...
+          <MuiLink
+            href="https://github.com/openfoodfacts/hunger-games/issues"
+            target="_blank"
+          >
+            {t("settings.reportIssue")}
+          </MuiLink>
+        </div>
+      </Stack>
+      <Divider sx={{ width: "100%" }} light />
+      <FooterWithLinks />
```
Comment (alexfauquette) [inline src/pages/settings/index.jsx:151]: Very nice

Maybe a slight improvement could be to move all the files you created to a folder `src/components/Footer` such that you can also reuse it in `src/pages/home`

**Comment by alexfauquette** [comment]
For the item 
> Add a link to share Hunger Games with friends, with a prepopulated explainer

I don't no. Maybe @teolemon have an idea

File: src/pages/settings/index.jsx (hunk 1 above)
Comment (Sudhanva-Nadiger) [reply src/pages/settings/index.jsx:151]: okayyyyy !!!! I will do that. Thanks <3

**Comment by Sudhanva-Nadiger** [comment]
@alexfauquette sir I have created another [pr](https://github.com/openfoodfacts/hunger-games/pull/539), don't know why that automatically get added to this pr .

**Pull request closed without merging**